packages/cli/src/output-manager.ts

```diff
@@ -0,0 +1,11 @@
+import { Output } from './util/output';
+
+let output = new Output(process.stderr, {
+  debug: false,
+  noColor: true,
+});
+
+/**
+ * A managed singleton instance of the Output class.
+ */
+export default output;
```

Comment:
I'm open to calling this something other than `output-manager` that makes it clear this is the singleton wrapper.

Reply:
> This allows other contexts to use the same singleton after the initial CLI logic parses the args and updates the values on the singleton.

Do you have an API in mind for that `mockClient` will do with output? Will it no longer initialize it within it's contstructor?

Reply:
I updated MockClient to use this